packages/web/src/lib/api.ts

```diff
@@ -2,17 +2,17 @@ import algosdk, { Algodv2, ALGORAND_MIN_TX_FEE, SuggestedParams } from "algosdk"
 
 import { ChainType, TxParams } from "../types";
 
-const mainNetClient = new Algodv2("", "https://algoexplorerapi.io", "");
-const testNetClient = new Algodv2("", "https://testnet.algoexplorerapi.io", "");
+const mainnetAlgoExplorer = new Algodv2("", "https://algoexplorerapi.io", "");
+const testnetAlgoExplorer = new Algodv2("", "https://node.testnet.algoexplorerapi.io", "");
```

Comment:
we should not declare them as global, please move inside the `algoexplorerAlgod` function